### PR TITLE
Cdodge/pass user name

### PR DESCRIPTION
**Pull request by chrisndodge**

### CODE REVIEW

**Comment by chrisndodge** [comment]
@afzaledx @muhhshoaib can you also review this?

File: edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html [hunk 1]
```diff
@@ -26,8 +26,12 @@
   </div>
 </div>
 <div class="footer-sequence border-b-0 padding-b-0">
-  <span> {% trans "Note: As soon as you finish installing and setting up the proctoring software,
-    you will be prompted to start your timed exam." %} </span>
+  <span>
+    {% blocktrans %}
+      Note: As soon as you finish installing and setting up the proctoring software,
+      you will be prompted to start your timed exam.
+    {% endblocktrans %}
```
Comment (chrisndodge) [inline edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html]: Note to reviewers, this is a drive by fix

File: edx_proctoring/backends/software_secure.py [hunk 2]
```diff
@@ -114,11 +112,25 @@ def pad(text):
...
+        callback_url = context['callback_url']
+        full_name = context['full_name']
+        first_name = ''
+        last_name = ''
+
+        if full_name:
+            name_elements = full_name.split(' ')
+            first_name = name_elements[0]
+            if len(name_elements) > 1:
+                last_name = ' '.join(name_elements[1:])
```
Comment (afzaledx) [inline edx_proctoring/backends/software_secure.py]: This would include the middle name as well. Is it intended?

**Comment by afzaledx** [comment]
:+1:

File: edx_proctoring/backends/software_secure.py (hunk 2 above)
Comment (chrisndodge) [inline edx_proctoring/backends/software_secure.py]: Yea, Software Secure's API only supports a first_name, last_name field. It's used by reviewers to compare the name that will be printed on the edX Certificate of completion to the photo ID that the user presented as part of their proctoring startup.

**Pull request merged**